app/models/journeyDomain/PreRequisitesDomain.scala

```diff
@@ -16,16 +16,18 @@
 
 package models.journeyDomain
 
-import config.Constants.DeclarationType._
+import config.Constants.DeclarationType.*
 import config.PhaseConfig
-import models.{OptionalBoolean, Phase}
+import models.journeyDomain.*
 import models.reference.Country
+import models.{OptionalBoolean, Phase}
 import pages.external.DeclarationTypePage
-import pages.preRequisites._
+import pages.preRequisites.*
 
 case class PreRequisitesDomain(
   ucr: Option[String],
   countryOfDispatch: Option[Country],
+  countryOfDestination: Option[OptionalBoolean],
```

Comment:
We can then remove `itemsDestinationCountry` and make countryOfDestination an `Option[Country]`